mlir/lib/Dialect/ArmSME/IR/ArmSME.cpp

```diff
@@ -12,6 +12,7 @@
 
 #include "mlir/Dialect/ArmSME/IR/ArmSME.h"
 #include "mlir/Dialect/LLVMIR/LLVMTypes.h"
+#include "mlir/IR/TypeUtilities.h"
```

Comment:
is this required for the `AllElementTypesMatch` constraint?

Reply:
Yep